changelog.txt

```diff
@@ -197,11 +197,11 @@ Changes in version Liquibase v4.18.0 (2022.11.30)
 - Use together for increased security and convenience of centralized properties
 
 * [PRO]Security
-- Vaults offer much greater security than defaults files or environment variables 
+- Vaults offer much greater security than defaults files orenvironment variables
```

Comment:
typo :)

Reply:
I saw that, but I din't change it so wasn't sure what it should be.

Reply:
I am assuming it is "for"